game/scripts/vscripts/components/points/game_length_vote.lua

```diff
@@ -47,4 +47,7 @@ function GameLengthVotes:SetGameLength ()
   end
   DebugPrint ( 'votes ' .. votes.short .. ', ' .. votes.normal .. ', ' .. votes.long .. ' result: ' .. length)
   GameRules.GameLength = length
+  CustomGameEventManager:Send_ServerToAllClients("gamelength_vote_confirmed", {
```

Comment:
You could move [this](https://github.com/OpenAngelArena/oaa/blob/master/game/scripts/vscripts/components/points/points.lua#L19-L31) to [here](https://github.com/OpenAngelArena/oaa/blob/master/game/scripts/vscripts/components/points/game_length_vote.lua#L49). This would be better either way.